app/controllers/reservations_controller.rb

```diff
@@ -511,6 +474,58 @@ def archive # rubocop:disable all
 
   private
 
+  def handle_new_reservation_errors
+    if can? :override, :reservation_errors
+      flash[:error] = 'Are you sure you want to continue? Please review '\
+        'the errors below.'
+      render :new
+    elsif AppConfig.check(:disable_requests)
+      flash[:error] = 'Please review the errors below.'
+      render :requests_disabled
+    else
+      flash[:error] = 'Please review the errors below. If uncorrected, '\
+        'any reservations with errors will be filed as a request, and '\
+        'subject to administrator approval.'
+      render :new_request
+    end
+  end
+
+  def handle_create_errors(errors)
+    case errors
+    when 'needs notes'
+      flash[:error] = 'Please give a short justification for this reservation'
+      @notes_required = true
+      @request_text = if AppConfig.get(:request_text).empty?
+                        'Please give a short justification for this '\
```

Comment:
this is a little out of scope but it seems like we should just set this as a default value for `request_text` rather than having this live in the controller